stratum/hal/lib/barefoot/bfrt_table_manager_test.cc

```diff
@@ -144,6 +144,22 @@ class BfrtTableManagerTest : public ::testing::Test {
 };
 
 constexpr int BfrtTableManagerTest::kDevice1;
+constexpr char kTableEntryText[] = R"pb(
+  table_id: 33583783
+      match {
+        field_id: 4
+        ternary {
+          value: "\211B"
+          mask: "\377\377"
+        }
+      }
+      action {
+        action {
+          action_id: 16783057
+        }
+      }
+      priority: 10
+)pb";
```

Comment:
We usually aim for the smallest possible scope when creating constants. In tests this would be the test class (`BfrtTableManagerTest`), as `static constexpr` member variables. Even though those variables are const and could be optimized away in most cases, they still need backing storage for uses involving a pointer. Hence we have code like `constexpr int BfrtTableManagerTest::kDevice1;` which creates the storage.